src/index.js

```diff
@@ -1,28 +1,117 @@
 import './index.css';
 
-const tasks = [
-  { description: 'Learn Webpack', completed: true, index: 0 },
-  { description: 'Finish quiz', completed: false, index: 1 },
-  { description: 'Learn supplication', completed: false, index: 2 },
-  { description: 'Wash utensils', completed: true, index: 3 },
-];
+let tasks = [];
+if (localStorage.getItem('tasks')) {
+  tasks = JSON.parse(localStorage.getItem('tasks'));
+}
+function saveTasks() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+function addTask(description) {
+  const task = {
+    description,
+    completed: false,
+    index: tasks.length,
+  };
+  tasks.push(task);
+  saveTasks();
```

Comment:
- [ ] Please note that new tasks should have their index set to the value of the length of the array. This means the index of the elements should start from 1, not zero. What do you think 🤔 ?
![index](https://user-images.githubusercontent.com/99656687/236139827-fd5f08c4-1954-4e10-899d-36415d93b047.png)